src/test/resources/allfields/input/instructions.csv

```diff
@@ -1,11 +1,11 @@
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,richard,,,,,,,,,,,,,,,,,,,,,,,,,,OPEN_ACCESS,,,,,,,,,,,,,user001,
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,joke,,,,,,,,,,,,,,,,,,,,2017-08-02,Date Copyrighted,,,,,OPEN_ACCESS,,,,,,,,,,,,,,
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,vesa,,,,,,,,,,,,,,,,,,,,,,,,,,OPEN_ACCESS,,,,,,,,,,,,,,
-ruimtereis03,Title for empty dataset,,,Donald,,Duck,,Duckstad,,,,,,,,,2016-07-30,unknown person,,This is an entry for a dataset with no data in it,empty space,,,,,,,,,,,,,,,,,,,,,,,GROUP_ACCESS,,D37000,,,,,,,,,,,user001,9ad0a3e4-39d4-47a0-b1a1-b6377ed516af
-ruimtereis04,Audio Dataset,,,Guus,,Geluk,,Duckstad,,,,,,,,,2016-07-31,,,This is a dataset with audio in it,,,,,,,,,,,,,,,,,,,,,audio/mpeg3,,,OPEN_ACCESS_FOR_REGISTERED_USERS,2016-07-30,D30000,path/to/a/random/sound/chicken.mp3,our daily wake up call,,dans,janvanmansum,Jans-test-files,continuous,,,,user001,773dc53a-1cdb-47c4-992a-254a59b98376
+DATASET     ,DC_TITLE                    ,DCT_ALTERNATIVE                      ,DCX_CREATOR_TITLES,DCX_CREATOR_INITIALS,DCX_CREATOR_INSERTIONS,DCX_CREATOR_SURNAME,DCX_CREATOR_DAI              ,DCX_CREATOR_ORGANIZATION    ,DCX_CREATOR_ROLE     ,DCX_CONTRIBUTOR_TITLES,DCX_CONTRIBUTOR_INITIALS,DCX_CONTRIBUTOR_INSERTIONS,DCX_CONTRIBUTOR_SURNAME,DCX_CONTRIBUTOR_DAI           ,DCX_CONTRIBUTOR_ORGANIZATION,DCX_CONTRIBUTOR_ROLE,DDM_CREATED,DCT_RIGHTSHOLDER,DC_PUBLISHER,DC_DESCRIPTION                                                                                                                                                                                                                                                                                                                   ,DC_SUBJECT ,DC_SUBJECT_SCHEME,DCT_TEMPORAL,DCT_TEMPORAL_SCHEME,DCT_SPATIAL,DCX_SPATIAL_SCHEME,DCX_SPATIAL_X,DCX_SPATIAL_Y,DCX_SPATIAL_NORTH,DCX_SPATIAL_SOUTH,DCX_SPATIAL_EAST,DCX_SPATIAL_WEST,DC_IDENTIFIER,DC_IDENTIFIER_TYPE       ,DCX_RELATION_QUALIFIER,DCX_RELATION_TITLE,DCX_RELATION_LINK,DCT_DATE   ,DCT_DATE_QUALIFIER,DC_TYPE,DC_FORMAT                                                              ,DC_LANGUAGE,DC_SOURCE                ,DDM_ACCESSRIGHTS                ,DDM_AVAILABLE,DDM_AUDIENCE,FILE_PATH                         ,FILE_TITLE                       ,FILE_ACCESSIBILITY,SF_DOMAIN,SF_USER     ,SF_COLLECTION  ,SF_PLAY_MODE,AV_FILE_PATH           ,AV_SUBTITLES                      ,AV_SUBTITLES_LANGUAGE,DEPOSITOR_ID,BASE_REVISION
```

Comment:
Does this actually work, with this extra formatting? That would be great for future development!

Reply:
In that case, also format the invalid `instructions.csv`

Reply:
It really works :)

Reply:
awesome! 🎉